volumegroup/README.md

```diff
@@ -78,6 +75,10 @@ message CreateVolumeGroupRequest {
   // This field is OPTIONAL. Refer to the `Secrets Requirements`
   // section on how to use this field.
   map<string, string> secrets = 3 [(csi.v1.csi_secret) = true];
+
+  // Specify volume_ids that will be added to the volume group.
+  // This field is OPTIONAL.
+  repeated string volume_ids = 4;
```

Comment:
there should be an error code and description that explains an incompatible volume_ids list is passed

Reply:
I was expecting an `INVALID_ARGUMENT` error or something new... If the volume can only be part of a single group, returning `ALREADY_EXISTS` and asking to use a different `name` is not a solution.

Reply:
i referred to https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolumegroupsnapshot-errors for it, i can add invalid if you prefer but single volume is a limitation from the SP, let me know if you still suggest to have invalid error

Reply:
I prefer `FAILED_PRECONDITION` for volumes that can not be grouped together, for whatever reason (already part of a group, different features, ...). The `ALREADY_EXISTS` can then be a clear error that a volumegroup with the same name already exists (and the caller can then check if the volumegroup actually has the expected volumes).

Reply:
@nixpanic PTAL